### PR TITLE
Vishxm do

### PR DESCRIPTION
**Pull request by vishxm**
Added 48th problem

### CODE REVIEW

**Comment by satyajitghana** [comment]
Hi, can you add the problem no/name in the heading of README ? and also add some documentation/code explanation ?

File: solutions/problem-006/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+```
```
Comment (satyajitghana) [inline solutions/problem-006/README.md]: add the problem name here

File: solutions/problem-006/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+n = 100
+t1 = 0.25*(n**4 + n**2 + 2*(n**3))
+t2 = (n*(n+1)*(2*n+1))/6
+
+print(int(t1-t2))
+```
```
Comment (satyajitghana) [inline solutions/problem-006/README.md:18]: add a bit of code documentation as well.
try to explain it in markdown also, the problem statement

**Review by satyajitghana — CHANGES_REQUESTED** [review]

File: solutions/problem-006/README.md (hunk 1 above)
Comment (vishxm) [reply solutions/problem-006/README.md]: Sure will do that

File: solutions/problem-006/README.md (hunk 2 above)
Comment (vishxm) [reply solutions/problem-006/README.md:18]: Can you check now, please?

**Review by satyajitghana — APPROVED** [review]

**Comment by satyajitghana** [comment]
Thanks for your contribution !

**Pull request merged**